BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -187,4 +194,71 @@ extension SideMenuFilterViewController {
 
     }
 
+    
+    func setUserPreferences() {
+        
+        defaultPreferences.set(distanceFilter, forKey: "distanceFilter")
```

Comment:
Keys should be namespaced with the app and the topic. For example:

`tech.blacksource.blacknectar.settings.filter.distance`. This prevents potential naming conflicts with other parts of the app.